### PR TITLE
WW 80

### PR DESCRIPTION
**Pull request by HoXuanHieu**
Update export data to Excel file

### CODE REVIEW

File: src/main/java/com/example/codeE/controller/TestController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    @GetMapping("/export")
+    public void testExportExcel() {
+        
+        var wb = new XSSFWorkbook();
+        ArrayList<User> list = new ArrayList<>();    
+        list.add(new User(UUID.randomUUID().toString(), "hieu", "hieu123@email.com", "de123233", "Student", "createdDate", "updatedDate"));
+        list.add(new User(UUID.randomUUID().toString(), "khiem", "khiem123@email.com", "de123123", "Student", "123-23-23", "2323-232-232"));
+        ExcelHelper.writeExcel(wb, list, "test.xlsx", "test");
+    }
+}
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/controller/TestController.java:27]: Why do you create a new API Endpoint just for testing? 
You should use an existing endpoint in the test module.

File: src/main/java/com/example/codeE/ulti/Constant.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.example.codeE.ulti;
+
+//all constant string will be declared here
+public class Constant {
+    public static final String FILE_PATH = "src/main/resources/excel/";
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/ulti/Constant.java:5]: This should not be here. What's the point of setting a file path, which is also pointing into the project, here?

File: src/main/java/com/example/codeE/ulti/excelHelper/ExcelHelper.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
+package com.example.codeE.ulti.excelHelper;
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/ulti/excelHelper/ExcelHelper.java:1]: Please change the path to util. "ulti" is not correct

**Review by th4tkh13m — CHANGES_REQUESTED** [review]
Please look at Nhan's testing structure.

File: src/main/java/com/example/codeE/ulti/excelHelper/ExcelHelper.java (hunk 3 above)
Comment (HoXuanHieu) [reply src/main/java/com/example/codeE/ulti/excelHelper/ExcelHelper.java:1]: will modified later

**Comment by HoXuanHieu** [comment]
because this branch has insecurity commit, We need to close pull request and delete this commit

**Pull request closed without merging**